--- a/MODULE.bazel
+++ b/MODULE.bazel
@@ -51,7 +51,7 @@
 
 http_archive(
     name = "org_gwtproject_gwt",
-    sha256 = "731879b8e56024a34f36b83655975a474e1ac1dffdfe72724e337976ac0e1749",
-    strip_prefix = "gwt-073679594c6ead7abe501009f8ba31eb390047fc",
-    url = "https://github.com/gwtproject/gwt/archive/073679594c6ead7abe501009f8ba31eb390047fc.zip",
+    sha256 = "48e60f2102becf84d435d26b55b60bf4b9455f528329a7ca12a856d9bc619d6f",
+    strip_prefix = "gwt-fbab97c0dc0af6c64c88a9bdbd8b63b525f6192f",
+    url = "https://github.com/aschleck/gwt/archive/fbab97c0dc0af6c64c88a9bdbd8b63b525f6192f.zip",
 )
