diff third_party/extern.bzl third_party/extern.bzl
index b715079..f842923 100644
--- third_party/extern.bzl
+++ third_party/extern.bzl
@@ -9,6 +9,5 @@ def extern(name, path = None):
         name = name,
         srcs = ["@com_google_javascript_closure_compiler//:externs"],
         outs = ["%s.js" % name],
-        tools = ["@bazel_tools//tools/jdk:jar"],
-        cmd = "$(location @bazel_tools//tools/jdk:jar) -xf  $(location @com_google_javascript_closure_compiler//:externs) %s; mv %s $@" % (path, path),
+        cmd = "unzip -p $(location @com_google_javascript_closure_compiler//:externs) %s > $@" % (path,),
     )
