diff --git jsinterop_generator.bzl jsinterop_generator.bzl
index 4895711..25ec26d 100644
--- jsinterop_generator.bzl
+++ jsinterop_generator.bzl
@@ -158,26 +158,17 @@ def _jsinterop_generator_impl(ctx):
     else:
         gwt_module_name = "%s.%s" % (ctx.attr.package_prefix, ctx.attr.gwt_module_name)
 
-    # format output
     arguments = [
         ctx.outputs._generated_jar.path,
         ctx.outputs._formatted_jar.path,
-        ctx.executable._google_java_formatter.path,
-        ctx.executable._jar.path,
     ]
-
-    tools = [
-        ctx.executable._google_java_formatter,
-        ctx.executable._jar,
-    ]
-
-    ctx.actions.run(
+    # Dumb no-op
+    ctx.actions.run_shell(
         inputs = [ctx.outputs._generated_jar],
-        tools = tools,
         outputs = [ctx.outputs._formatted_jar],
-        executable = ctx.executable._format_jar_script,
         progress_message = "Formatting java classes",
         arguments = arguments,
+        command = "cp -L $1 $2",
     )
 
     return [
@@ -209,11 +196,6 @@ _jsinterop_generator = rule(
         "gwt_module_name": attr.string(),
         "runtime_deps": attr.label_list(),
         "custom_preprocessing_pass": attr.string_list(),
-        "_jar": attr.label(
-            cfg = "exec",
-            executable = True,
-            default = Label("@bazel_tools//tools/jdk:jar"),
-        ),
         "_google_java_formatter": attr.label(
             cfg = "exec",
             executable = True,
