diff -Naur local_java_repository.bzl local_java_repository.bzl 
--- local_java_repository.bzl
+++ local_java_repository.bzl
@@ -98,7 +98,7 @@
     native.toolchain(
         name = "bootstrap_runtime_toolchain_definition",
         target_settings = [":%s_settings_alias" % name],
-        toolchain_type = "@bazel_tools//tools/jdk:bootstrap_runtime_toolchain_type",
+        toolchain_type = "@rules_java//toolchains:bootstrap_runtime_toolchain_type",
         toolchain = runtime_name,
     )
 
@@ -232,3 +232,4 @@
     """
     _local_java_repository_rule(name = name, java_home = java_home, version = version, build_file = build_file)
     native.register_toolchains("@" + name + "//:runtime_toolchain_definition")
+    native.register_toolchains("@" + name + "//:bootstrap_runtime_toolchain_definition")
